src/content/developers/docs/scaling/plasma/index.md

```diff
@@ -35,5 +35,6 @@ Multiple projects provide implementations of Plasma that you can integrate into
 ## Further reading {#further-reading}
 
 - [EthHub on Plasma](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/plasma/)
+- [A COMPLETE GUIDE TO ETHEREUM PLASMA](https://www.blockchain-council.org/blockchain/a-complete-guide-to-ethereum-plasma/)
```

Comment:
@MrBrain295 Thanks for dropping this =)... 
```suggestion
- [A Complete Guide To Ethereum Plasma](https://www.blockchain-council.org/blockchain/a-complete-guide-to-ethereum-plasma/)
```
Could we switch this to title casing?

Reply:
I thought about that they do use that casing in the article but I’m fine with either.